Android/ru/strings.xml

```diff
@@ -685,7 +685,7 @@
     <string name="setting_section_preferences">Кастомизация</string>
     <string name="setting_section_tools">Инструменты</string>
     <string name="widget_unable_to_update">Не удалось обновить виджет</string>
-    <string name="event_show_details">Показать детали</string>
```

Comment:
Please translate it as "Show Details (English)" because the detail page is only in English

Reply:
Ok. Only for Android?

Reply:
both